com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs

```diff
@@ -49,6 +49,25 @@ internal bool UseAddressablesInEditor
 #endif
             }
         }
+        
+        internal bool ClearAllAddressablesCache
+        {
+            get
+            {
+#if UNITY_EDITOR
+                return UnityEditor.EditorPrefs.GetBool($"{Name}_user-clear-all-addressable-cache", true);
```

Comment:
Can we, please, set the default value of this property to `false`?
Thus, we can prevent forced cache cleanup on build machines.